src/components/Accordion/_Accordion.scss

```diff
@@ -3,15 +3,36 @@
     color: inherit;
     width: 100%;
 
-    &__button {
+    input {
+        position: absolute;
+        opacity: 0;
+        z-index: -1;
+
+        &:checked {
+            + .accordion__label .icon--minus {
+                display: inline;
+            }
+            + .accordion__label .icon--plus {
+                display: none;
+            }
+            ~ .accordion__content {
+                display: inline-block;
+            }
+        }
+    }
+
+    &__label {
         @include button--outline;
 
-        align-items: center;
         border: none;
-        display: inline-flex;
-        font-weight: 500;
         justify-content: space-between;
-        width: 100%;
+        font-weight: 500;
+        cursor: pointer;
+        align-items: center;
```

Comment:
Craziest nit but your linter should be bothering you that this should be the first property based on our linting rules—is it not telling you that? https://github.com/NYPL/nypl-design-system/blob/development/.stylelintrc.json#L41

Reply:
I installed `stylelint` in my vscode so that error shows up now. It looks like stylelint is not setup in the project correctly because that error should come up when running `npm test`.

Although I fixed it for this file, there are errors in other files. This should be a separate ticket not just to fix those files but to make that linter part of the entire process.

I ran `npx stylelint "**/*.scss"` and got this output:

<img width="1337" alt="Screen Shot 2020-12-01 at 1 03 28 PM" src="https://user-images.githubusercontent.com/1280564/100779064-cfa9e280-33d5-11eb-9365-0d20f3b6a5b0.png">

Reply:
Oh nice, guess I was only getting those from my editor directly—so lot of stuff was getting missed.

I've filed #450 to track this.